src/raster/mod.rs

```diff
@@ -4,7 +4,9 @@ mod rasterband;
 mod types;
 mod warp;
 
-pub use rasterband::{Buffer, ByteBuffer, RasterBand};
+pub use rasterband::{
+    get_color_interpretation_by_name, get_color_interpretation_name, Buffer, ByteBuffer, RasterBand,
```

Comment:
i don't think we need to pub use the methods since we pub use RasterBand. However, i guess we need to pub use GDALColorInterp?

Reply:
I'm a bit confused here. `get_color_interpretation_{by_}name` are not `RasterBand`'s methods (and I'm not sure if it is possible to pub use struct's methods), so pub using `RasterBand` doesn't make these functions visible. And `rasterband` module is private, so I need to pub use these functions here.

Reply:
you are right. somehow i thought they are Rasterband methods.